go.mod

```diff
@@ -4,6 +4,7 @@ go 1.17
 
 require (
 	github.com/CosmWasm/wasmd v0.28.0
+	github.com/InjectiveLabs/injective-core v1.7.1
```

Comment:
Where do we need injective-core pkg? We should use copied ore types from sdk-go itself

Reply:
Oh yes you're right. updated